nsv13/code/game/machinery/blast_shower.dm

```diff
@@ -0,0 +1,86 @@
+/obj/machinery/shower/blast_shower
+	name = "blast shower"
+	desc = "The RS-455. High pressure shower system guaranteed to remove radiation at unprecedented rates."
+	icon = 'nsv13icons/obj/blast_shower.dmi'
+	icon_state = "blast_shower"
+	density = FALSE
+	use_power = NO_POWER_USE
```

Comment:
How does it generate the pressure for blasting if it doesn't use power?

Reply:
Most showers use gravity from a header tank to produce pressure a rate that is comfortable to stand in.

This is intended on being at a vastly higher pressure than that.

Discussion has been had with author re: no power, power and emag states.

Reply:
The blast shower is powered by your radiation output, that's how it knows when to stop too :)